code/modules/jobs/job_types/roguetown/youngfolk/squire.dm

```diff
@@ -8,67 +8,148 @@
-	else
-		pants = /obj/item/clothing/under/roguetown/tights
-		shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/guard
-		armor = /obj/item/clothing/suit/roguetown/armor/chainmail
-		shoes = /obj/item/clothing/shoes/roguetown/boots
-		belt = /obj/item/storage/belt/rogue/leather
-		beltr = /obj/item/storage/belt/rogue/pouch
-		beltl = /obj/item/storage/keyring/guardcastle
-		backr = /obj/item/storage/backpack/rogue/satchel
-		if(H.mind)
-			H.mind.adjust_skillrank(/datum/skill/combat/maces, 1, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/combat/bows, 1, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 2, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/combat/knives, 1, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/misc/athletics, 1, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
-			H.mind.adjust_skillrank(/datum/skill/misc/riding, 2, TRUE)
-			H.change_stat("strength", 1)
-			H.change_stat("perception", 1)
-			H.change_stat("constitution", 1)
-			H.change_stat("speed", 1)
+/datum/outfit/job/roguetown/squire
+	shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/guard
+	shoes = /obj/item/clothing/shoes/roguetown/boots
+	belt = /obj/item/storage/belt/rogue/leather
+	beltl = /obj/item/storage/keyring/guardcastle
+
+/datum/job/roguetown/squire/after_spawn(mob/living/L, mob/M, latejoin = TRUE)
+	. = ..()
+	if(ishuman(L))
+		var/mob/living/carbon/human/H = L
+		H.advsetup = 1
+		H.invisibility = INVISIBILITY_MAXIMUM
+		H.become_blind("advsetup")
+
+/datum/advclass/squire/lancer
+	name = "Lancer Trainee"
+	tutorial = "A hopeful for the next generation of knightly mounted lancers and infantry pike specialists, \
+	your training with polearms sets you apart from other squires."
+	outfit = /datum/outfit/job/roguetown/squire/lancer
+		
+	category_tags = list(CTAG_SQUIRE)
+
+/datum/outfit/job/roguetown/squire/lancer/pre_equip(mob/living/carbon/human/H)
+	r_hand = /obj/item/rogueweapon/spear
+	armor = /obj/item/clothing/suit/roguetown/armor/chainmail
+	neck = /obj/item/clothing/neck/roguetown/chaincoif
+	pants = /obj/item/clothing/under/roguetown/chainlegs/iron
+	gloves = /obj/item/clothing/gloves/roguetown/leather
+	wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+	backr = /obj/item/storage/backpack/rogue/satchel
+	backpack_contents = list(
+		/obj/item/storage/belt/rogue/pouch,
+	)
+	if(H.mind)
+		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
```

Comment:
like i said- literally just. polearms 3 back to 2 and swords 3 to 2 and i think you can do literally whatever else you want skillwise, give em or give em back whatever. statwise that's a pretty good spread but if u wanna throw some extra int on it (bc they're...studying, or whatever? idfk) i don't think anyone would blink twice

up to other folk now, i am stepping out since i elaborated here, do what you will